nengo_loihi/splitter.py

```diff
@@ -139,6 +146,8 @@ def place_nodes(networks):
     # Only ChipReceiveNodes can be run on chip
     for node in networks.original.all_nodes:
         if isinstance(node, ChipReceiveNode):
+            # TODO: This if branch can never happen as ChipReceiveNodes
```

Comment:
The reason I put this here is in case we or someone manually makes a `ChipReceiveNode` in the original network. It's exactly the kind of thing that we start off saying "oh, no one would do that." Then later "oh, someone did that, huh." Then even later "oh, I want to do that."

Reply:
Fair enough.  Although I still think a comment in there to this effect would be helpful -- when I was reading through the code I was confused as I thought this was an expected branch to have happen.  :)

Reply:
(and I should have caught this during my review of the original PR, but I misinterpretted what was happening then and didn't realize this was happening before ChipReceiveNodes were created)